### PR TITLE
Do some refactoring

### PR DESCRIPTION
**Pull request by vvval**

### CODE REVIEW

File: src/Specification/Filter/Map.php [hunk 1]
```diff
@@ -73,22 +73,21 @@ public function getFilters(): array
...
      * @inheritDoc
      */
-    public function getValue()
+    public function getValue(): array
     {
-        $result = [];
-        foreach ($this->filters as $name => $filter) {
-            $result[$name] = $filter;
+        foreach ($this->filters as $filter) {
+            return $filter->getValue();
```
Comment (wolfy-j) [inline src/Specification/Filter/Map.php]: invalid

File: src/Specification/Value/EnumValue.php [hunk 2]
```diff
@@ -29,8 +29,10 @@ final class EnumValue implements ValueInterface
      */
     public function __construct(ValueInterface $base, ...$values)
     {
+        sort($values);
```
Comment (wolfy-j) [inline src/Specification/Value/EnumValue.php]: ?

Comment (vvval) [reply src/Specification/Value/EnumValue.php]: Moved to paginator

File: src/Specification/Filter/Map.php (hunk 1 above)
Comment (vvval) [reply src/Specification/Filter/Map.php]: Removed return type hint

**Pull request merged**